geometry_msgs/Pose pose
sensor_msgs/JointState joint_state
string[] ignore
---
bool collision_free

